sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -42,6 +43,15 @@ const Lottery = (props: { listing: Listing }) => {
   const { listingsService, profile } = useContext(AuthContext)
   const { data } = useFlaggedApplicationsMeta(listing?.id)
   const duplicatesExist = data?.totalPendingCount > 0
+  const shouldExpireData = !profile?.userRoles.isAdmin && !process.env.lotteryDaysTillExpiry
+  let formattedExpiryDate: string
+  if (!process.env.lotteryDaysTillExpiry) {
```

Comment:
Could def be misunderstanding, but this seems like: if the env variable is falsey, then we use it?

Reply:
Was a mistake, originally meant to put `!!` but isn't really necessary here. Corrected 👍